pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+filterwarnings =
+    ignore::UserWarning
```

Comment:
nit: nl

Reply:
Also, I caution against blanket ignores like this.  https://docs.pytest.org/en/latest/warnings.html#pytest-mark-filterwarnings might work for what you're doing.